config/locales/views/export/en.yml

```
@@ -0,0 +1,27 @@
+en:
+  export:
+    download_submissions:
+      title: Download submissions
+      choose_exercises: From which exercises do you want to download submissions?
+      choose_series: From which series do you want to download submissions?
+      choose_courses: From which courses do you want to download submissions?
+      next_step: Next step
+      start_download: Start download
+      choose_options: Select any extra options you wish to apply to your download.
+      only_last_submission_label: Only include the last submission?
+      only_last_submission_label_help: If checked, only the last submission from a student for an exercise will be included. Otherwise, all submissions will be downloaded.
+      deadline_label: Take the deadline into account?
+      deadline_label_help: If checked, only submissions before the deadline will be downloaded. Otherwise, even submissions sent in after the deadline will be present.
+      group_submissions_label: How to group submissions in the download?
+      group_by_student_help: Result contains a folder per student containing their latest submission per exercise.
```

```suggestion
      group_by_student_help: Result contains a folder per student containing their submission(s) per exercise.
```
There is an option to not only have the latest submission per exercise, right?